src/main/java/edu/kis/powp/jobs2d/command/DriverCommand.java

```diff
@@ -5,7 +5,7 @@
 /**
  * DriverCommand interface.
  */
-public interface DriverCommand {
+public interface DriverCommand extends Cloneable {
```

Comment:
I suggest resigning from Cloneable interface, Clone method and associated exceptions.